tools/tests/s3tests-runner.sh

```diff
@@ -175,7 +176,7 @@ _run() {
 
   export S3TEST_CONF="${TMPDIR}/s3tests.conf"
   export S3_USE_SIGV4=ON
-  if pytest \
+  if timeout --preserve-status "${S3TEST_TIMEOUT}" pytest \
```

Comment:
Would it make sense to add a result "timeout"?

Reply:
Yes, done.